src/main/java/io/mo/db/ConnectionManager.java

```diff
@@ -42,7 +42,7 @@ public static Connection getConnection(){
                 }
                 return connections[0];
             } catch (SQLException e) {
-                LOG.error("The mo-tester can not get valid conneciton from mo, and will wait 10 seconds and retry...");
+                LOG.error("The mo-tester can not get valid conneciton from mo with[user="+userName+", pwd="+pwd+"], and will wait 10 seconds and retry...");
```

Comment:
This function seems to be duplicated with #L61, it equals to getConnection(0)

Reply:
it is a problem, I need to fix